src/ansible_creator/resources/common/patterns/sample_pattern/playbooks/site.yml

```diff
@@ -0,0 +1,69 @@
+---
+- name: Weather forecast
+  hosts: localhost
+  gather_facts: false
+  tasks:
+    - name: Collections requirements
+      ansible.builtin.debug:
+        msg: |
+          Warning: This sample pattern relies on the ansible.scm, ansible.utils and community.general collections.
+          Please manually add these to the collection's requirements file to use the sample pattern as-is.
```

Comment:
This shouldn't be necessary if using the pattern EE.

Reply:
yeah, this is an add to an exisiting collection which suggests the parent collection should now depend on these collections since the pattern is contained within the collection, so if a new EE was built with this collection the resulting EE would not work for the added pattern. We talkaed about this, but with the add of a sample pattern, we didn't want to modify the users's collection requirements since what they probably want is an example structure and probably won't keep the tasks that require ansible.scm and community general. That said, if the user does want to try the pattern, we wanted to bring attention to the fact these deps were not explicetly added to the parent collect.